README.md

```diff
@@ -49,3 +49,28 @@ Once you have verified that the `protoc` binary is in your `$PATH`, you can
 build `hbbft` using cargo:
 
     $ cargo build [--release]
+
```

Comment:
@DrPeterVanNostrand: Can you please add the formatting guidelines here. You have applied a coding standard to the code. Please share that standard with us.